src/tests/ftest/util/soak_utils.py

```diff
@@ -32,10 +35,11 @@
 from mdtest_utils import MdtestCommand
 from oclass_utils import extract_redundancy_factor
 from pydaos.raw import DaosApiError, DaosSnapshot
-from run_utils import run_remote
+from run_utils import run_local, run_remote
 from test_utils_container import add_container
 
 H_LOCK = threading.Lock()
+id_counter = count(start=1)
```

Comment:
Suggestion - you might want to start with a larger number so the job ids are easily sortable (if you need that). E.g. with `start=` you'll get
```
1
2
...
9
10  <-- throws off sorting / printing ids in a list format
```

But e.g. `start=1000`, numbers will be aligned from 1000 to 9999
```
1001
1002
...
1009
1010
...
9999
10000 # <-- finally not aligned
```

Reply:
I don't think I need that because the id is added into the log names or it is used in a dictionary

Reply:
fwiw I agree with Dalton, especially if the `id_counter` is used in file names.